includes/features/SU-SWS/stanford_courses/stanford_courses.feature

```diff
@@ -3,26 +3,26 @@ Feature: Stanford Courses
   As an administrative user
   I want to ensure that the Stanford Courses module is working properly
 
-  @api
+  @api @safe
   Scenario: Verify Stanford Courses is enabled
     Given I am logged in as a user with the "administrator" role
     And I am on "/admin/modules"
     Then the checkbox "Stanford Courses" should be checked
 
-  @content @live
+  @live
```

Comment:
Because we can expect to see "Current Courses" when the site is deployed, launched, and live, the tag should be @safe.